fixtures/regressions/logging-callback-interface/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "uniffi-fixture-logging-callback-interface"
 edition = "2021"
+authors = ["Firefox Sync Team <sync-team@mozilla.com>", "Viktor Jansson <v.l.jansson@gmail.com>"]
```

Comment:
Please let me know if you want me to revert this change. I git blamed and saw that Ben has contributed previously, let me know if you want me to add him as well (I think Mark is a member of the sync team?).

Reply:
Ben's also in the sync team - we'd also be up for removing this line entirely, but until then I think this is fine.